.gitignore

```diff
@@ -20,3 +20,4 @@ man
 .channel
 dist/
 .venv/
+.idea/
```

Comment:
Leftover?

Reply:
Should I remove it? It's for the IntelliJ/PyCharm project files I don't want to commit into repository

Reply:
No I guess we can keep it, I wonder why it is not possible to configure PyCharm to not put file in the repository but rather un `~/.idea/project_name`